assemblyline/odm/models/alert.py

```diff
@@ -39,53 +42,56 @@ def __lt__(self, other: DetailedItem) -> bool:
+@odm.model(index=True, store=False, description="""Stores information about screenshots taken during the analysis of the file. Each screenshot has a name, description, and the hashes of the image and its thumbnail, offering a visual reference that can aid in manual review processes.
+""")
 class Screenshot(odm.Model):
-    name = odm.Keyword(description="Name of the screenshot")
-    description = odm.Keyword(description="Description of the screenshot")
-    img = odm.SHA256(description="SHA256 hash of the image")
-    thumb = odm.SHA256(description="SHA256 hash of the thumbnail")
+    name = odm.Keyword(description="The name or title of the screenshot.")
+    description = odm.Keyword(description="A brief description of the screenshot's content.")
+    img = odm.SHA256(description="The SHA256 hash of the full-size screenshot image.")
+    thumb = odm.SHA256(description="The SHA256 hash of the thumbnail version of the screenshot.")
 
     def __hash__(self) -> int:
         return hash(tuple(sorted(self.as_primitives().items())))
 
 
-@odm.model(index=True, store=False, description="Assemblyline Detailed result block")
+@odm.model(index=True, store=False, description="""Provides a comprehensive breakdown of specific attributes and their associated analysis results.
+""")
 class DetailedResults(odm.Model):
-    attack_pattern = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed Att&ck patterns")
-    attack_category = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed Att&ck categories")
-    attrib = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed attribution")
-    av = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed AV hits")
-    behavior = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed behaviors for the alert")
-    domain = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed domains")
-    heuristic = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed heuristics")
-    ip = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed IPs")
-    uri = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed URIs")
-    yara = odm.List(odm.Compound(DetailedItem), default=[], description="List of detailed YARA rule hits")
+    attack_pattern = odm.List(odm.Compound(DetailedItem), default=[], description="Detailed information on MITRE ATT&CK® framework patterns identified in the analysis.")
+    attack_category = odm.List(odm.Compound(DetailedItem), default=[], description="Detailed information on MITRE ATT&CK® framework categories associated with the alert.")
+    attrib = odm.List(odm.Compound(DetailedItem), default=[], description="Detailed attribution information that provide context by suggesting associations with known malware families, suspected threat actors, or ongoing campaigns.")
```

Comment:
Detailed attribution information that provides context by suggesting associations with known malware families, etc. etc.

Reply:
Yup, singular here.  Good catch.